src/sentry/static/sentry/app/views/organizationStream/overview.jsx

```diff
@@ -254,9 +254,9 @@ const OrganizationStream = createReactClass({
     const currentQuery = this.props.location.query || {};
     if ('cursor' in currentQuery) {
       requestParams.cursor = currentQuery.cursor;
-    } else if (NEW_FILTERS_TEST in currentQuery) {
+    } else if (ConfigStore.get('user').isSuperuser) {
```

Comment:
Could you make it so that I can disable the new parser via a query param? Is still sometimes useful for visual testing

Reply:
Done